3rdparty/everest/CMakeLists.txt

```diff
@@ -11,6 +11,19 @@ target_include_directories(everest
           include/everest/kremlib
           ${MBEDTLS_DIR}/library/)
 
+# Pass-through MBEDTLS_CONFIG_FILE and MBEDTLS_USER_CONFIG_FILE
```

Comment:
It seems wrong and error-prone that we're doing this in each 3rdparty subdirectory. We'll need to copy-paste this whenever we add a new 3rdparty library, and it's likely we'll forget.

Reply:
There is a better solution which was implemented in the new PSA repo, but it would be a *much* more major change than this. I would suggest doing it this way for now, and making a ticket to port Ronald's PSA solution back to MbedTLS asap.